Bottom: f8f17b82c419faecd9b22becb86ff382f8f8f4a8
Top:    0199e27e644692ae14cae0e7d49411add70813a1
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-24 15:34:16 +0800

autofs-5.0.7 - fix master map mount options matching

The master map options pattern matching is fairly primitive since it
doesn't need to be very sophisticated. The current mount option pattern
matching can't match mount options with quotes or embedded colons and so
it can't pass these options through as a global options string.

But it must be able to match a fairly large class of strings, including
the above case, so they can be passed through as global options. Of
course it can't try and validate them since it can't know what mount type
they may be used with.


---

diff --git a/CHANGELOG b/CHANGELOG
index 503a21e..7a6d52d 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -52,6 +52,7 @@
 - add changlog entry for coverity fixes.
 - fix probe each nfs version in turn for singleton mounts.
 - misc man page fixes.
+- fix master map mount options matching.
 
 25/07/2012 autofs-5.0.7
 =======================
